libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/CloudAdapter.cs

```diff
@@ -25,6 +26,8 @@ namespace Microsoft.Bot.Builder.Integration.AspNet.Core
     /// </summary>
     public class CloudAdapter : CloudAdapterBase, IBotFrameworkHttpAdapter
     {
+        private readonly ConcurrentDictionary<Guid, StreamingActivityProcessor> _streamingConnections = new ConcurrentDictionary<Guid, StreamingActivityProcessor>();
```

Comment:
Though this won't affect consumers of the SDK, the value should be [`IStreamingActivityProcessor`](https://github.com/microsoft/botbuilder-dotnet/blob/0e332233cca3b725696c783cb6561cd94a14475b/libraries/Microsoft.Bot.Builder/Streaming/IStreamingActivityProcessor.cs#L13).

Let's future proof it in case there's a decision to open up this `ConcurrentDictionary` to developers.

Reply:
The methods I am calling are in the StreamingActivityProcessor, so in order to use the interface, these methods need to be exposed in the interface as well. I am not sure if we want to expose the use case specific methods I added to be on the interface since they are not generic enough to be at that level.

Reply:
I lowkey kind of hate that this dictionary has to exist. I wish there were a way to hide it behind a neat abstraction such that we could replace it with a more intelligent solution later without taking dependency on passing around nebulous streaming classes like "StreamingActivityProcessor"